tests/features/bootstrap/FeatureContext.php

```diff
@@ -386,6 +386,42 @@ public function assertFieldOptionSelected($option, $select) {
   }
 
   /**
+   * Checks that a certain radio input is checked in a specific field.
+   *
+   * @param string $radio
+   *   The label of the radio input to find.
+   * @param string $field
+   *   The label of the field the radio is part of.
+   *
+   * @throws \Exception
+   *   Thrown when the field or the radio is not found, or if the radio is not
+   *   checked.
+   *
+   * @Then the radio :radio from field :field should be checked
+   */
+  public function assertFieldRadioSelected($radio, $field) {
```

Comment:
Ok, then it has been already done with said commit by you on previous round of QA 😄  (see latest revision [here](https://github.com/ec-europa/joinup-dev/pull/382/files#diff-a2ccc714735c13b3a710a340994f3c74R405)).